RNTester/js/AccessibilityExample.js

```diff
@@ -133,6 +133,73 @@ class AccessibilityExample extends React.Component {
   }
 }
 
+class AccessibilityActionsExample extends React.Component {
+  render() {
+    return (
+      <View>
+        <RNTesterBlock title="Non-touchable with activate action">
+          <View
+            accessible={true}
+            accessibilityActions={[{name: 'activate'}]}
+            onAccessibilityAction={event => {
+              switch (event.nativeEvent.actionName) {
+                case 'activate':
+                  Alert.alert('Alert', 'View is clicked');
+                  break;
+              }
+            }}>
+            <Text>Click me</Text>
+          </View>
+        </RNTesterBlock>
+
+        <RNTesterBlock title="View with multiple actions">
+          <View
+            accessible={true}
+            accessibilityActions={[
+              {name: 'cut', label: 'cut label'},
+              {name: 'copy', label: 'copy label'},
+              {name: 'paste', label: 'paste label'},
```

Comment:
I like this idea, but not a big fan of overloading the prop on the js side. It's a bit error prone, due to special 'name' definitions that don't appear outside the iOS client code, logic that could break if label is accidentally added/omitted when needed.

Specifically, from the js side, it's not clear what the behavior would be if someone defines one of the special case actions with a label accidentally or what if someone omits label from a custom action definition.

an example:
```
accessibilityActions={[
  {name: 'activate', label: 'tap'}, // this would add an extra custom action in addition to defining "accessibilityActivate" behavior on iOS
  {name: 'cut'}, // could we just assume that the label would also be `cut` here for their custom action? right now this would just be dropped completely on iOS without warning 
]}
```

it would be awesome if we could have a prop for the strongly typed events like "activate", "escape", etc, and a separate prop for custom actions specifically.

Reply:
I think keeping these as a single prop is nice, as it keeps all actions together, regardless if their implementation details on iOS/Android, especially considering that iOS and Android have different predefined actions. 

That being said, maybe we should have two different types allowed here.  One that is a "predefined action" such as "Activate", which doesn't let you set the label, and one that is a "custom action", which requires you to set both a name and a label.